### PR TITLE
OEL-1159: Align templates with modified search variant of card pattern

### PR DESCRIPTION
**Pull request by donquixote**
https://citnet.tech.ec.europa.eu/CITnet/jira/browse/OEL-1159

### CODE REVIEW

File: modules/oe_showcase_search/templates/node--search-result.html.twig [hunk 1]
```diff
@@ -7,18 +7,22 @@
 {% set _title %}
   <a class="text-underline-hover" href="{{ url }}">{{ label }}</a>
```
Comment (escuriola) [inline modules/oe_showcase_search/templates/node--search-result.html.twig]: Not related, but text-underline-hover must be changed to standalone class

File: modules/oe_showcase_search/templates/node--search-result.html.twig [hunk 2]
```diff
@@ -7,18 +7,22 @@
...
 {% endset %}
+{% set _badges = [] %}
+{% for _item in content.field_category|field_value %}
+  {% set _badges = _badges|merge([{
+    label: _item,
```
Comment (escuriola) [inline modules/oe_showcase_search/templates/node--search-result.html.twig:13]: badges are links, and links need to add the class standalone, if not, you'll see an underline: 
image

**Review by escuriola — CHANGES_REQUESTED** [review]

File: modules/oe_showcase_search/templates/node--search-result.html.twig (hunk 2 above)
Comment (donquixote) [reply modules/oe_showcase_search/templates/node--search-result.html.twig:13]: In the old version there were no links for these category items.
I am pushing a fix that reproduces that old behavior without links.

File: modules/oe_showcase_search/templates/node--search-result.html.twig (hunk 1 above)
Comment (donquixote) [reply modules/oe_showcase_search/templates/node--search-result.html.twig]: Fixed.

**Review by escuriola — DISMISSED** [review]

**Review by drishu — APPROVED** [review]

**Review by escuriola — APPROVED** [review]

**Pull request merged**